tests/x509/x509-req-test.sh

```diff
@@ -129,7 +129,7 @@ run_success "req -new -key ./certs/server-key.pem -config ./test-prompt.conf -ou
 run_success "req -text -in tmp.csr"
 SUBJECT=`./wolfssl req -in tmp.csr -text | grep -A1 "X509v3 Subject Alternative Name"`
 EXPECTED="        X509v3 Subject Alternative Name: 
-            email:facts@wolfssl.com, Registered ID:1.1.1.1, Registered ID:surname, URI:facts@wolfssl.com"
+            email:facts@wolfssl.com, URI:facts@wolfssl.com, Registered ID:surname, Registered ID:1.1.1.1"
```

Comment:
Could you pull this change out into a separate PR? I noticed this during the release cycle also and saw that you had a fix here. I think it should be separate of the feature additions.

Reply:
Disregard this. The fix was also done in this PR https://github.com/wolfSSL/wolfCLU/pull/177 which got merged in.